src/components/Projects/AddProject/AddProject.jsx

```diff
@@ -28,12 +37,12 @@ const AddProject = props => {
         type="text"
         className="form-control"
         aria-label="New Project"
-        placeholder="Project Name (required) type to add."
+        placeholder="Project Name (required)"
         onChange={e => changeNewName(e.target.value)}
```

Comment:
please try to add 
```value = {newName} ``` 
to reset the value in the input box after creating a project